src/App.js

```diff
@@ -0,0 +1,21 @@
+import { Routes, Route } from 'react-router-dom';
+import Nav from './components/Nav';
+import Home from './components/Home';
+import Categories from './components/Categories';
+import Books from './components/Books';
+
+const App = () => { //eslint-disable-line
+
+  return (
+    <div className="app">
+      <Nav />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/books" element={<Books />} />
+        <Route path="/categories" element={<Categories />} />
```

Comment:
- [ ] Grea job you made three routes but as required you should just make two routes one for the book lists and the form to add a new book and another for categories conatin button to check status so kindly remove the home and keep the books to contain book list and form to add new book and categoreies. 📌🚀🔥💯

![image](https://user-images.githubusercontent.com/58553711/200139146-073a595b-8181-493e-be31-c5e8980fe66d.png)